frontends/p4/typeChecking/typeChecker.cpp

```diff
@@ -2835,7 +2903,11 @@ void TypeInference::checkCorelibMethods(const ExternMethod* em) const {
 
 const IR::Node* TypeInference::postorder(IR::MethodCallExpression* expression) {
     if (done()) return expression;
+    auto method = expression->to<IR::MethodCallExpression>()->method;
```

Comment:
`expression` is already an `IR::MethodCallExpression`. Please move after log message.

Reply:
ok.